ground/src/main/java/com/google/android/ground/model/task/Condition.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.model.task
+
+/** The task ID. */
+typealias TaskId = String
+/** The selected option ID for each task. */
+typealias OptionId = String
+/** The selected options keyed by task ID. */
+typealias TaskSelections = Map<TaskId, Set<OptionId>>
+
+/**
+ * Describes a user-defined condition on a task, which determines whether the given task should be
+ * hidden due to failure of fulfillment based on the input expressions.
+ */
+data class Condition
+@JvmOverloads
```

Comment:
Do we still need this annotation? I think this is no-op as we don't have any java codebase left.

Reply:
Compiled and ran fine, so removed in both places!